packages/validators/src/validators.ts

```diff
@@ -105,11 +106,24 @@ export const isHash = (hash: any): hash is Hash =>
  *
  * @method isTransactionHash
  *
- * @param {string} hash
+ * @param {Hash|Transaction} input
  *
  * @return {boolean}
  */
-export const isTransactionHash = (hash: any): hash is Hash => isTrytesOfExactLength(hash, HASH_SIZE)
+export const isTransactionHash = (input: Hash | Transaction, mwm?: number): boolean => {
```

Comment:
We could change the signature to accept attached transaction trytes. This way we don't need to import `transaction-converter` eliminating circular dependency.

Reply:
Should be good to break it down to two methods:
- `isTransactionHash`
- `isTransactionTrytes`

That way based on trytes length, can simply check trailing zero count. This would make `mwm` a required parameter.

Reply:
Looks as the way to go! I still think that `mwm` should be optional, to be able to validate transaction trytes that haven't been attached.